src/main/java/org/benetech/servicenet/domain/RequestLogger.java

```diff
@@ -0,0 +1,193 @@
+package org.benetech.servicenet.domain;
+import java.util.Objects;
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.Lob;
+import javax.persistence.Table;
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+
+import java.io.Serializable;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+/**
+ * A RequestLogger.
+ */
+@Entity
+@Table(name = "request_logger")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+public class RequestLogger implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(
+        name = "UUID",
+        strategy = "org.hibernate.id.UUIDGenerator"
+    )
+    private UUID id;
+
+    @Column(name = "request_uri")
+    private String requestUri;
+
+    @Column(name = "remote_addr")
+    private String remoteAddr;
+
+    @Column(name = "request_method")
+    private String requestMethod;
+
+    @Lob
+    @Type(type = "org.hibernate.type.TextType")
+    @Column(name = "request_parameters", columnDefinition = "clob")
+    private String requestParameters;
+
+    @Lob
+    @Type(type = "org.hibernate.type.TextType")
+    @Column(name = "request_body", columnDefinition = "clob")
+    private String requestBody;
```

Comment:
Do we really want to save all request and response bodies? I imagine this could take A LOT of space after some time

Reply:
Request body is not that bad. The problem might be with response body.

Reply:
Now there is save for response body if the status code is different than 200.